index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <p>Hello World, we are software developers!</p>
+    <header id ="header"> 
+      Samar
+      <img id="header-img" src="https://concepto.de/wp-content/uploads/2019/09/plantas-e1569789174411.jpg">  
+      <nav id="nav-bar">
+        <a href="" class="nav-link">https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-projects/build-a-product-landing-page</a>
+        <a href="" class="nav-link">https://product-landing-page.freecodecamp.rocks/</a>
+        <a href="" class="nav-link">https://microverse.pathwright.com/library/microverse-trials-collaborative-project-32595/68742/path/step/51183551/</a>
+      </nav>
+      <video src="https://www.youtube.com/watch?v=d9wXEisXqco&ab_channel=PakistanCricket" id="video"></video>
+      <form action="https://www.freecodecamp.com/email-submit" id="form"></form>
+      <input type="email" placeholder="enter email" id="email">
+      <input type="submit" id="submit">
+    </header>
```

Comment:
This is mainly the content of your webpage. It would be a good choice if you enclose it under the `<section>` or `<article>`tag instead of the `<header>` tag for your content.